pkg/network/protocols/common_test.go

```diff
@@ -0,0 +1,43 @@
+package protocols
```

Comment:
I don't see the value of duplicating the logic in 2 different locations
I do suggest to unify the logic in a single instance or duplicate the tests for both packages

Reply:
move to 2 instances of the tests as pkg/trace is an independent package ; we should not include tracer in system-probe